Makefile

```diff
@@ -614,13 +614,13 @@ scalaclean:
 	(cd $(ROOTDIR)/scala-package && mvn clean)
 
 scalapkg:
-	(cd $(ROOTDIR)/scala-package && mvn install -DskipTests)
+	(cd $(ROOTDIR)/scala-package && mvn install -q -DskipTests)
```

Comment:
This change will make it hard to troubleshoot PR failure.
What's the limitation of the log out?
Can we redirect maven output the a log file, and make the log file as artifact to download?

Reply:
We don't store artifacts right now. I didn't find a way to suppress the log output less than with -q

Won't a failure be printed in any case? On a failing PR the -q can be removed for diagnosis anyway.

Reply:
I assume, developer has tested locally with maven already. If there is a failure in CI, in most cases, error message is not enough to troubleshoot a PR failure.

Switching -q flag to debug on CI isn't a right approach, and it's very time consuming effort.

Reply:
Can you put examples of what you describe or provide an alternative? Why removing -q on a PR wouldn't be enough to troubleshoot any failures?  That PR would repro the error with full verbosity